FoodOrderingSystem.py

```diff
@@ -0,0 +1,700 @@
+            session.close()
+
+    def view_orders(self, order_id, session):
+        """ View particular details of an order """
+        try:
+            view = session.query(FoodCategory, FoodDetails, CustOrderSelection, CustomerDetails, CustOrderStatus).\
+                filter(CustomerDetails.cust_id == CustOrderStatus.cust_id).\
+                filter(CustOrderSelection.food_id == FoodDetails.food_id).\
+                filter(FoodCategory.category_id == FoodDetails.category_id).\
+                filter(CustOrderSelection.order_id == CustOrderStatus.order_id).\
+                filter(CustOrderStatus.order_id == order_id)
+                # label('total_price'), func.sum(CustOrderSelection.food_qty*FoodDetails.price)
+                # (func.sum(CustOrderSelection.food_qty*FoodDetails.price).label("total_price"))
+            return view
+        except Exception as ex:
+            print("Error getting order, error={}".format(str(ex)))
+        finally:
+            session.close()        
+
+    def view_order_status(self, order_id, session):
+        """ View status of a particular order """
+
+        try:
+            view = session.query(FoodCategory, FoodDetails, CustOrderSelection, CustomerDetails, CustOrderStatus, DeliveryPerson).\
+                filter(CustomerDetails.cust_id == CustOrderStatus.cust_id).\
+                filter(CustOrderSelection.food_id == FoodDetails.food_id).\
+                filter(FoodCategory.category_id == FoodDetails.category_id).\
+                filter(CustOrderSelection.order_id == CustOrderStatus.order_id).\
+                filter(DeliveryPerson.delivery_person_id == CustOrderStatus.delivery_person_id).\
+                filter(CustOrderStatus.order_id == order_id)
+                # label('total_price'), func.sum(CustOrderSelection.food_qty*FoodDetails.price)
+                # (func.sum(CustOrderSelection.food_qty*FoodDetails.price).label("total_price"))
+            return view
+        except Exception as ex:
+            print("Error getting order, error={}".format(str(ex)))
+        finally:
+            session.close()
+
+    def view_revenue_today(self, order_status, session):
+        """ View revenue of today """
+        # Sum all the total price of the delieverd orders
+
+        try:
+            revenue_today = session.query(FoodDetails, CustOrderSelection, CustomerDetails, CustOrderStatus).\
+            filter(CustomerDetails.cust_id == CustOrderStatus.cust_id).\
+            filter(CustOrderSelection.order_id == CustOrderStatus.order_id).\
+            filter(CustOrderSelection.food_id == FoodDetails.food_id).\
+            filter(CustOrderStatus.order_status == order_status).\
+            filter(CustOrderStatus.checkout_time <= datetime.today())
+            # label('total_price'), func.sum(CustOrderSelection.food_qty*FoodDetails.price)
+            # (func.sum(CustOrderSelection.food_qty*FoodDetails.price).label("total_price"))
+            return revenue_today
```

Comment:
view_revenue_today() is wrong! It should return the amount in numnbers, like the sum of all the prices of all the orders today. and same for view_revenue_month() for the entire month instead.